src/cms/views/pages/page_actions.py

```diff
@@ -112,48 +111,33 @@ def delete_page(request, page_id, region_slug, language_code):
     )
 
 
-@login_required
-def copy_short_url(request, page_id, region_slug, language_code):
+def expand_short_url(request, short_url_id):
     """
-    Creates short url of page and copies url to clipboard.
+    Searches for a page with requested short_url_id and redirects to that page.
     """
-    region = Region.get_current_region(request)
-    page = get_object_or_404(region.pages, id=page_id)
-    page_translation = page.get_translation(language_code)
-
-    short_url = f"{request.scheme}://{request.get_host()}/{page_translation.short_url}"
-    try:
-        pyperclip.copy(short_url)
-        messages.success(
-            request,
-            _("URL '{}' was successfully copied to clipboard.".format(short_url)),
-        )
-    # pylint: disable=broad-except
-    except Exception as e:
-        logger.exception(e)
-        messages.warning(
-            request,
-            _("URL '{}' could not be copied to clipboard.".format(short_url)),
-        )
+    queryset = PageTranslation.objects.filter(short_url_id=short_url_id)
+    page_translation = queryset.first()
 
-    return redirect(
-        "pages",
-        **{
-            "region_slug": region_slug,
-            "language_code": language_code,
-        },
-    )
+    if (
+        page_translation
+        and page_translation.latest_public_revision
+        and page_translation.page.archived == False
```

Comment:
I think pylint wants to see this:
```suggestion
        and not page_translation.page.archived
```